test/k8sT/DatapathConfiguration.go

```diff
@@ -151,6 +172,14 @@ var _ = Describe("K8sDatapathConfig", func() {
 		})
 	})
 
+	Context("Transparent encryption with IPv4Only", func() {
+		It("Check connectivity with transparent encryption enabled and IPv6 disabled", func() {
+			deployCilium("cilium-ds-patch-ipv4-only-ipsec.yaml")
```

Comment:
shouldn't this be skipped in flannel case?

Reply:
Skipped for now will debug why this is not working and add it back later.